hack/istio/skupper/install-skupper-demo.sh

```diff
@@ -303,9 +289,8 @@ minikube_install_skupper() {
   while ! ${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n ${MYSQLSKUPPERNS} get svc mysqldb-v1 &> /dev/null ; do echo -n '.'; sleep 1; done; echo
   SKUPPER_MYSQL_IP="$(${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n ${MYSQLSKUPPERNS} get svc mysqldb-v1 -o jsonpath='{.spec.clusterIPs[0]}')"
   infomsg "MySQL IP over the Skupper link: ${SKUPPER_MYSQL_IP}"
-  infomsg "Configuring Bookinfo ratings-v2-mysql to talk to MySQL over the Skupper link via the ServiceEntry"
-  ${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n bookinfo set env deployment/ratings-v2-mysql MYSQL_DB_HOST="${SKUPPER_MYSQL_IP}"
-  ${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n bookinfo patch serviceentry mysqldb.test --type='json' -p='[{"op": "replace", "path": "/spec/addresses/0", "value": "'${SKUPPER_MYSQL_IP}'"}, {"op": "replace", "path": "/spec/endpoints/0/address", "value": "'${SKUPPER_MYSQL_IP}'"}]'
+  infomsg "Configuring Bookinfo ratings-v2-mysql to talk to MySQL over the Skupper link"
+  ${CLIENT_EXE} --context ${CLUSTER1_ISTIO} -n bookinfo set env deployment/ratings-v2-mysql MYSQL_DB_URL="mysqldb://${SKUPPER_MYSQL_IP}:3306/mysql"
```

Comment:
Does this actually work?  There is no MYSQL_DB_URL when you install bookinfo - just the host which is what the original script uses.
```
$ kubectl get pods -n bookinfo -l app=ratings -l version=v2-mysql -ojsonpath={..spec.containers[0].env} |jq
[
  {
    "name": "DB_TYPE",
    "value": "mysql"
  },
  {
    "name": "MYSQL_DB_HOST",
    "value": "mysqldb"
  },
  {
    "name": "MYSQL_DB_PORT",
    "value": "3306"
  },
  {
    "name": "MYSQL_DB_USER",
    "value": "root"
  },
  {
    "name": "MYSQL_DB_PASSWORD",
    "value": "password"
  }
]
```